### ISSUE TITLE
WebDriverAgent failed to launch on IOS 13.3.1

### ISSUE THREAD

**Issue by mparimi06** [appium/appium#13969]
## The problem

WebDriverAgent failed to launch on IOS 13.3.1 when running from appium, can able to build without any errors using Xcode.
Same works fine when iphone was not upgraded with version 13.3

## Environment

* Appium version (or git revision) that exhibits the issue:   V1.16.0 and V1.15.1
* Last Appium version that did not exhibit the issue (if applicable):
* Desktop OS/version used to run Appium: MACOS 10.15.2
* Node.js version (unless using Appium.app|exe): 
* Npm or Yarn package manager:
* Mobile platform/version under test: iPhoneXR , ios 13.3.1
* Real device or emulator/simulator: Real Device
* Appium CLI or Appium.app|exe: Tried with both 

## Details

Please find below logs, issue started only after iphone upgraded to latest version.

## Link to Appium logs

https://gist.github.com/mparimi06/6d13988155f62cf3225ce047138dfdbb

**Comment by mykola-mokhnach** [comment]
Duplicate of https://github.com/appium/appium/issues/13927

**Comment by chenzhiyou** [comment]
how to fix this problem
